views/partials/user-list-single.ejs

```diff
@@ -0,0 +1,25 @@
+<tr>
+    <td>
+        <%= user._id %>
+    </td>
+    <td>
+        <%= user.username %>
+    </td>
+    <td>
+        <%= user.birthdate %>
+    </td>
+    <td>
+        <%= user.email %>
+    </td>
+    <td>
+        <div class="btn-group" role="group" aria-label="action">
+            <button type="button" id="view" data-user-id="<%= user._id %>" class="btn btn-primary">
+                <i class="fa-sharp fa-solid fa-pen-to-square"></i>
+            </button>
+            <button type="button" id="delete" data-user-id="<%= user._id %>" class="btn btn-primary">
```

Comment:
delete button should have a danger color because it is after all a dangerous operation

There is an existing bootstrap class system for that documented in [this link](https://getbootstrap.com/docs/5.3/components/buttons/#variants)

I think the delete button should have the danger color for color semantics.